src/components/Cards/Cards.js

```diff
@@ -15,29 +16,35 @@ export default function Cards({onBookmark, exercises, filter}) {
     }
   });
   return (
-    <div>
-      <>
-        {filteredExercises.length === 0 ? (
-          <p>
-            Du hast noch keine Favoriten gespeichert. Lass dich weiter inspirieren oder atme einfach ein paar mal tief
-            durch.
-          </p>
-        ) : (
-          filteredExercises.map(exercise => (
-            <Card
-              onBookmark={onBookmark}
-              bookmarked={exercise.bookmarked}
-              id={exercise.id}
-              key={exercise.id}
-              category={exercise.category}
-              description={exercise.description}
-              repeat={exercise.repeat}
-              duration={exercise.duration}
-              pathname={exercise.pathname}
-            />
-          ))
-        )}
-      </>
-    </div>
+    <StyledWrapper>
+      {filteredExercises.length === 0 ? (
+        <p>
+          Du hast noch keine Favoriten gespeichert. Lass dich weiter inspirieren oder atme einfach ein paar mal tief
+          durch.
+        </p>
+      ) : (
+        filteredExercises.map(exercise => (
+          <Card
+            onBookmark={onBookmark}
+            bookmarked={exercise.bookmarked}
+            id={exercise.id}
+            key={exercise.id}
+            category={exercise.category}
+            title={exercise.title}
+            description={exercise.description}
+            repeat={exercise.repeat}
+            duration={exercise.duration}
+            pathname={exercise.pathname}
```

Comment:
Just to test something different: I think you can also use destructuring here as well:
```suggestion
        filteredExercises.map(({bookmarked, id, category, title, description, repeat, duration, pathname}) => (
          <Card
            onBookmark={onBookmark}
            bookmarked={bookmarked}
            id={id}
            key={id}
            category={category}
            title={title}
            description={description}
            repeat={repeat}
            duration={duration}
            pathname={pathname}
```